src/components/Button/Button.tsx

```diff
@@ -339,26 +317,31 @@ const Button: VibeComponent<ButtonProps, unknown> & {
 
     if (success) {
       return (
-        <button {...buttonProps}>
-          {successIcon ? (
-            <Icon
-              iconType={Icon?.type.ICON_FONT}
-              clickable={false}
-              icon={successIcon}
-              iconSize={successIconSize}
-              className={cx({
-                [styles.leftIcon]: !!successText
-              })}
-              ignoreFocusStyle
-            />
-          ) : null}
-          {successText}
+        <button {...buttonProps} key={`${id}-success`}>
+          <span className={styles.successContent}>
+            {successIcon ? (
+              <Icon
+                iconType={Icon?.type.ICON_FONT}
+                clickable={false}
+                icon={successIcon}
+                iconSize={successIconSize}
+                className={cx({
+                  [styles.leftIcon]: !!successText
+                })}
+                ignoreFocusStyle
+              />
+            ) : null}
+            {successText}
+          </span>
+          <span aria-hidden className={styles.textPlaceholder}>
+            {children}
```

Comment:
Used this text placeholder to keep the success button the same size, same technique that was used in the loading state, it saves us the need to use the resize observer

Reply:
I have a comment about it in the CSS file